cassandane/tiny-tests/JMAPEmail/email_query_sieve_regression

```diff
@@ -0,0 +1,250 @@
+#!perl
+use Cassandane::Tiny;
+
+sub test_email_query_sieve_regression
+  : needs_component_sieve : NoMunge8Bit : RFC2047_UTF8 {
+  my ($self)  = @_;
+  my $jmap    = $self->{jmap};
+  my $imap    = $self->{store}->get_client();
+  my $uuidgen = Data::UUID->new;
```

Comment:
[Just a drive by comment, didn't notice at first that this is still in draft]

The spaces between the `:` and the attribute names are jarring.  I'm a little surprised it even works, which is interesting.  I've only ever seen them written without the space, and had assumed that was important.

Our style wants the `{` on its own line, and four space indents (not two).  Was this run through perltidy or something?

Reply:
Yes, I used perltidy. I am not aware of any particular style guide that we have for Perl in cyrus, and if we do I would prefer it to be documented or preferably automated. That being said, I'll manually fix the points you raised.

Reply:
Updated accordingly.